build_tools/ci/run_matmul_test.sh

```diff
@@ -88,7 +95,9 @@ fi
 if [ -z "${3-}" ]; then
   PEANO=/opt/llvm-aie
 else
-  PEANO=`realpath "$3"`
+  if [ -d "$PEANO" ]; then
```

Comment:
```suggestion
  if [ "$PEANO" ]; then
```

Same for XRT_DIR and VITIS

Because if the user provided an invalid path, it would be good to fail informatively